### PR TITLE
Tweak a few more items in the network request article.

### PR DESCRIPTION
**Pull request by jpmedley**
@oliverdunk, I double-checked this to make sure we had caught everything. I found two items you should look at. Ignore the writerly fixes I through in.

Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

-
-
-

### CODE REVIEW

File: site/en/docs/extensions/mv3/network-requests/index.md [hunk 1]
```diff
@@ -73,15 +73,15 @@ non-secure HTTP access to a given host or set of hosts, it must declare the perm
...
 ## Security considerations {: #security-considerations }
 
-### Avoiding cross-site scripting vulnerabilities {: #xss }
+### Avoid cross-site scripting vulnerabilities {: #xss }
 
 When using resources retrieved via `fetch()`, your extension service worker should be careful not to
-fall victim to [cross-site scripting][9]. Specifically, avoid using dangerous APIs such as the
-below:
+fall victim to [cross-site scripting][9]. Specifically, avoid using dangerous APIs such as
+`innerHTML` or `eval()`. For example:
```
Comment (oliverdunk) [inline site/en/docs/extensions/mv3/network-requests/index.md:84]: Nit: I probably wouldn't mention eval here given it isn't possible in MV3.

File: site/en/docs/extensions/mv3/network-requests/index.md [hunk 2]
```diff
@@ -143,6 +144,7 @@ chrome.runtime.sendMessage(
               encodeURIComponent(request.itemId)},
     response => parsePrice(response.text()));
```
Comment (oliverdunk) [inline site/en/docs/extensions/mv3/network-requests/index.md:145]: It looks like the newline between these two blocks is no longer rendering. Is there a way to fix that?

**Review by oliverdunk — APPROVED** [review]
LGTM with nits, thanks! 👍

File: site/en/docs/extensions/mv3/network-requests/index.md (hunk 1 above)
Comment (jpmedley) [reply site/en/docs/extensions/mv3/network-requests/index.md:84]: I didn't say "don't use it". I said "avoid it". Isn't that valid?

Comment (jpmedley) [reply site/en/docs/extensions/mv3/network-requests/index.md:84]: I misread your statement.

Comment (jpmedley) [reply site/en/docs/extensions/mv3/network-requests/index.md]: Done.

**Review by AmySteam — APPROVED** [review]
LGTM 👍

**Pull request merged**